src/networking.c

```diff
@@ -1232,14 +1238,48 @@ int clientHasPendingReplies(client *c) {
 
 /* Return true if client connected from loopback interface */
 int islocalClient(client *c) {
+    struct ifaddrs *ifa, *addrs;
+    struct sockaddr_in *s4;
+    struct sockaddr_in6 *s6;
+    void *in_addr;
+    char loip[NET_IP_STR_LEN + 1];
```

Comment:
In other places in Redis, a char array of size NET_IP_STR_LEN without `+ 1` is used. I guess the nul terminator is included already. WDYT?

Reply:
@zuiderkwast Thanks for reviewing. The `NET_IP_STR_LEN` valued 46 includes the null terminator. The code is updated. https://github.com/redis/redis/pull/11664/commits/8290daa89aa04abd242f09f4840b48ad170b0594